src/tally.F90

```diff
@@ -1230,20 +1247,21 @@ subroutine score_general_mg(p, t, start_index, filter_index, i_nuclide, &
           else
             score = p % last_wgt
           end if
+
           if (i_nuclide > 0) then
-            score = score * nucxs % get_xs('inv_vel', p_g, UVW=p_uvw) / &
-                 matxs % get_xs('total', p_g, UVW=p_uvw) * flux
+            score = score * nucxs % get_xs('inverse-velocity', p_g, UVW=p_uvw) &
+                 / nucxs % get_xs('total', p_g, UVW=p_uvw) * flux
```

Comment:
Well this is an analog estimator, and so a collision was sampled based on the total macroscopic xs.  So shouldnt we be factoring out the total xs probability (since the sampling effectively incorporates that)?  I suggest you check to make sure the inverse velocity scores produced by your new code produce consistent and sensical answers compared to the other estimators/nuclidic or not/survival biasing or not estimators.  I had all those comparisons running when I originally set these score lines.